build/BUILD

```diff
@@ -21,23 +22,31 @@ filegroup(
     tags = ["automanaged"],
 )
 
+# ensure /etc/nsswitch.conf exists so go's resolver respects /etc/hosts
+container_image(
```

Comment:
can you set `mode = 0644` too? currently it's adding `nsswitch.conf` as executable.

Reply:
done